.github/workflows/nightly_publish.yml

```diff
@@ -123,8 +123,9 @@ jobs:
         run: |
           ./gradlew :integration:test "-Dai.djl.default_engine=PyTorch"
           ./gradlew :integration:clean
-          ./gradlew :integration:test "-Dai.djl.default_engine=OnnxRuntime"
-          ./gradlew :integration:clean
+          # OnnxRuntime 1.17.1 requires GLIBC 2.27
```

Comment:
Is there another OS we can use when testing with ARM?

GH self-hosted runners will have arm64 machines soon, maybe we just wait for that and we can move away from our self hosted arm64 runners and avoid all these GLIBC version issues